doc/classes/PackedScene.xml

```diff
@@ -106,7 +106,7 @@
 	<members>
 		<member name="_bundled" type="Dictionary" setter="_set_bundled_scene" getter="_get_bundled_scene" default="{ &quot;conn_count&quot;: 0, &quot;conns&quot;: PackedInt32Array(), &quot;editable_instances&quot;: [], &quot;names&quot;: PackedStringArray(), &quot;node_count&quot;: 0, &quot;node_paths&quot;: [], &quot;nodes&quot;: PackedInt32Array(), &quot;variants&quot;: [], &quot;version&quot;: 3 }">
 			A dictionary representation of the scene contents.
-			Available keys include "rnames" and "variants" for resources, "node_count", "nodes", "node_paths" for nodes, "editable_instances" for base scene children overrides, "conn_count" and "conns" for signal connections, and "version" for the format style of the PackedScene.
+			Available keys include "rnames" and "variants" for resources, "node_count", "nodes", "node_paths" for nodes, "editable_instances" for base scene child overrides, "conn_count" and "conns" for signal connections, and "version" for the format style of the PackedScene.
```

Comment:
Again grammar

Reply:
I did mean "the several overrides each of different children" because the "overrides" here refers to the property overrides for scene inheritance.

Yeah this description is _awful_.

Reply:
That's not what it means though, the "editable_instances" refer to a list of nodes which are editable

Reply:
Improved the description here